components/table/table-wrapper.js

```diff
@@ -1,14 +1,84 @@
 import '../colors/colors.js';
 import '../scroll-wrapper/scroll-wrapper.js';
 import { css, html, LitElement, nothing } from 'lit';
+import { getComposedParent, isComposedAncestor } from '../../helpers/dom.js';
 import { cssSizes } from '../inputs/input-checkbox.js';
-import { getComposedParent } from '../../helpers/dom.js';
 import { getFlag } from '../../helpers/flags.js';
+import { isInteractiveInComposedPath } from '../../helpers/interactive.js';
 import { PageableMixin } from '../paging/pageable-mixin.js';
 import ResizeObserver from 'resize-observer-polyfill/dist/ResizeObserver.es.js';
 import { SelectionMixin } from '../selection/selection-mixin.js';
 
 const colSyncFix = getFlag('GAUD-8228-8186-improved-table-col-sync', true);
+const prefersReducedMotion = window.matchMedia('(prefers-reduced-motion: reduce)').matches;
+
+const SELECTORS = {
```

Comment:
Do we have this problem anywhere sticky headers/controls are used? For instance, list controls?